Lib/dis.py

```diff
@@ -245,7 +245,7 @@ def show_code(co, *, file=None):
 _ExceptionTableEntry = collections.namedtuple("_ExceptionTableEntry",
     "start end target depth lasti")
 
-_OPNAME_WIDTH = 20
+_OPNAME_WIDTH = 25
```

Comment:
We could avoid needing to update this manually, right?

```suggestion
_OPNAME_WIDTH = max(map(len, opmap))
```